contrib/fwupd.spec.in

```diff
@@ -369,6 +369,7 @@ mkdir -p --mode=0700 $RPM_BUILD_ROOT%{_localstatedir}/lib/fwupd/gnupg
 %{_libdir}/fwupd-plugins-3/libfu_plugin_optionrom.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_pci_bcr.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_pci_mei.so
+%{_libdir}/fwupd-plugins-3/libfu_plugin_pci_pch.so
```

Comment:
I think this should probably be named pci_dci no?  Unless you're thinking to get other attributes from this later too.

Reply:
I think the PCH is a very interesting device with a lot more to measure.